src/Atquin.EntityFramework.Logging/EntityFrameworkLogger.cs

```diff
@@ -111,7 +111,7 @@ public void Log(LogLevel logLevel, int eventId, object state, Exception exceptio
 
             try
             {
-                db.SaveChanges();
+                db.SaveChangesAsync();
```

Comment:
is this really a good idea without returning the task or doing an async await?  what happens on an exception?